homematic/rootfs/opt/hm/bin/createBackup.sh

```diff
@@ -68,8 +68,9 @@ if [[ -d "${TMPDIR}" ]]; then
   cp -a /data/* "${TMPDIR}/usr/local/etc/config/"
 
   # move crRFD files to data sub-dir to be compatible
+  rm -rf "${TMPDIR}/usr/local/etc/config/crRFD"
   mkdir -p "${TMPDIR}/usr/local/etc/config/crRFD/data"
-  mv "${TMPDIR}/usr/local/etc/config/crRFD/*" "${TMPDIR}/usr/local/etc/config/crRFD/data/" 2>/dev/null || true
+  cp -r /data/crRFD/. "${TMPDIR}/usr/local/etc/config/crRFD/data/"
```

Comment:
Why not simply use the following instead:

```
cp -a /data/crRFD "${TMPDIR}/usr/local/etc/config/crRFD/data"
```

This should work aswell and looks cleaner to me.

Reply:
And then you have to alter the "mkdir -p" call one line above to

```
mkdir -p "${TMPDIR}/usr/local/etc/config/crRFD"
```

Reply:
ok, I've updated my branch

Reply:
Ok, thanks. Looks better now. So please retest with your environment and then let use know if the backup+restore procedure still works as expected.

Reply:
ok, I've now tested:
- update backup script using docker exec
- create backup (no error in ui)
- import backup into new install of raspberrymatic
=> i can see and control homematic ip devices

Reply:
perfect. and what about old/traditional BidCos/homematic devices? do you see them?

Reply:
I don't have any of those, sorry

Reply:
ok.